Linked list/Removing an Item

```diff
@@ -0,0 +1,49 @@
+** REMOVING AN ITEM **
+
+class Node:
+   def __init__(self, data=None):
+      self.data = data
+      self.next = None
+class SLinkedList:
+   def __init__(self):
+      self.head = None
+
+   def Atbegining(self, data_in):
+      NewNode = Node(data_in)
+      NewNode.next = self.head
+      self.head = NewNode
+
+# Function to remove node
+   def RemoveNode(self, Removekey):
+      HeadVal = self.head
+         
```

Comment:
# checking for remove a node

Reply:
not for me add a comment inside the code and make file .py